plugin-interface/src/main/scala/mesosphere/marathon/plugin/RunSpec.scala

```diff
@@ -25,6 +25,16 @@ trait RunSpec {
     * The networks that this run specification will join.
     */
   val networks: Seq[NetworkSpec]
+
+  /**
+    * Volume definitions
+    */
+  val volumes: Seq[VolumeSpec]
+
+  /**
+    * Volume mounts.
+    */
+  val volumeMounts: Seq[VolumeMountSpec]
```

Comment:
Because it is just the beginning of unifying apps and pods. Otherwise, in order to get rid of all tiny idiosyncrasies it would be necessary to finishing their unification in addition to the volume refactoring and the pod PV implementation, which is way out of scope.